package.json

```diff
@@ -1,24 +1,31 @@
 {
```

Comment:
this part is actually enforced by npm itself - whenever you run an npm command that alters the package.json, it changes the indent back to 2 spaces (which is a style from the node.js core).